### DISCUSSION TITLE
Borrowing from external data that is misaligned w.r.t. the visit schedule

### DISCUSSION THREAD

**Discussion by chstock** [openpharma/brms.mmrm#85] [Ideas]
As discussed recently, when borrowing from external data that is misaligned w.r.t. the visit schedule, linear (additive) mixed models might be helpful to derive priors for MMRMs. In practice, this challenge could occur quite frequently and might be something to be addressed in a future vignette.

The attached file includes a first (very preliminary) exploration of models with spline-based smooths fitted using `brms::brm()` and `mgcv::gam()`, and marginal means computed with the `{emmeans}` package.

[Bayesian linear (additive) mixed model with spline-based smooths.pdf](https://github.com/openpharma/brms.mmrm/files/14397897/Bayesian.linear.additive.mixed.model.with.spline-based.smooths.pdf)

Related to https://github.com/openpharma/brms.mmrm/issues/53.

**Comment by yonicd** [comment]
Small clarification question.

Am I reading correctly that the true treatment effect (PBO-TRT) is

```
> fev_data2 <- mutate(
+ fev_data2,
+ FEV1 = ifelse(AVISIT == "VIS1" & ARMCD == "TRT", FEV1 + 2, FEV1), #2
+ FEV1 = ifelse(AVISIT == "VIS2" & ARMCD == "TRT", FEV1 + 6, FEV1), #4
+ FEV1 = ifelse(AVISIT == "VIS3" & ARMCD == "TRT", FEV1 + 8, FEV1), #8
+ FEV1 = ifelse(AVISIT == "VIS4" & ARMCD == "TRT", FEV1 + 10, FEV1), #12
+ FEV1 = ifelse(AVISIT == "VIS5" & ARMCD == "TRT", FEV1 + 12, FEV1) #16
+ )
```

and the estimated from both methods is

week | brms Est. | brms LCL | brms UCL | mgcv Est | mgcv LCL | mgcv UCL | Diff Est. | Diff LCL | Diff UCL
---|---|---|---|---|---|---|---|---|---|
2 | −7.77 | −10.93 | −4.36 | −7.86 | −11.14 | −4.59 | 0.09 | 0.20 | 0.23
4 | −9.68 | −12.46 | −6.80 | −9.72 | −12.56 | −6.89 | 0.04 | 0.10 | 0.09
8 | −13.09 | −16.13 | −9.93 | −13.11 | −16.22 | −10.00 | 0.03 | 0.09 | 0.07
12 | −15.69 | −18.82 | −12.80 | −15.71 | −18.70 | −12.72 | 0.02 | −0.12 | −0.08
16 | −17.62 | −21.04 | −14.06 | −17.66 | −21.21 | −14.11 | 0.04 | 0.17 | 0.05

Why are the estimated contrasts much bigger than the true effect?

(I probably misread something in the beginning that I am not seeing)

**Comment by chstock** [comment]
> Why are the estimated contrasts much bigger than the true effect?
 
> (I probably misread something in the beginning that I am not seeing)

Thanks for having had a look, @yonicd, an sorry for the confusion. I didn't add a table with the simulated differences (which I do not know) or the observed ones, which would probably have made things clearer. The `fev_data` already assumes a treatment effect, and on top of this difference I added what is shown in the `mutate()` function above. The idea was just to create an example with very obviously different and non-linear mean trajectories in the two arms. What is shown doesn't yet allow to compare truth (or observed values) and estimation.

**Comment by wlandau** [comment]
Converting this excellent thread to a discussion as it describes a use case of #40 (as opposed to a built-in package feature).
